tests/unit_tests/data_steward/utils/participant_summary_requests_test.py

```diff
@@ -159,6 +172,36 @@ def test_get_deactivated_participants(self,
 
         self.assertEqual(expected_response, dataset_response)
 
+    @mock.patch('utils.participant_summary_requests.store_participant_data')
+    @mock.patch(
+        'utils.participant_summary_requests.get_site_participant_information')
+    def test_get_site_participant_information(
+        self, mock_get_site_participant_information,
+        mock_store_participant_data):
+
+        # Pre conditions
+        mock_get_site_participant_information.return_value = self.fake_participant_info_dataframe
+
+        # Tests
+        dataframe_response = psr.get_site_participant_information(
+            self.project_id, self.fake_hpo)
```

Comment:
Right here, you are storing a mocked object and using it to issue other calls.  If you are testing `get_site_participant_information` you:
1.  shouldn't mock the call to the function you are testing.  you should mock calls that function might make(e.g. to the Participant Summary API).
2. You should check the value of the dataframe it returns is what you expect.  You should not be testing the `store_participant_data` function again.  It should have already been tested.

Mocked objects will almost never fail an assertion, because they are mocks.